### ISSUE TITLE
Streamflix provider not working while website works fine

### ISSUE THREAD

**Issue by ob5cur3** [stantanasi/streamflix#89]
### Description

Streamflix provider shows an error that "No source found" but there are at least two sources available in the website directly and able to play the stream without any issues. In the app it shows the two sources and whichever source I choose it gives the same error.

### Reproduction steps

1. Open Streamflix App
2. Choose SFlix provider
3. Choose any of the available video in there (e.g. Damsel or Origin)
4. Click Watch and choose any source (VidCloud or Upcloud)

### App version

1.5.1

### Operating system

Fire TV Stick 4K Max - Fire OS 7.6.6.9

### Please confirm the following

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] If related to a provider, I have checked the site and it works, but not the app.

**Comment by stantanasi** [comment]
The videos servers have change their encryption method, as soon as I find a solution I will made a new release.

**Comment by stantanasi** [comment]
No, this is not fixed in the new version 1.5.2
Currently I'm working on app support for mobile and then I'll implement a new provider (probably SoraStream). So, until there is a definitive solution for the SFlix provider, I recommend you to use AnyMovie

**Comment by intellimediatv** [comment]
Gotcha i will wait on Sorastream its a beast of a provider lol Any Movie provider does not have a lot of content as Streamflix did

**Comment by intellimediatv** [comment]
AnyMovie is also not working at the moment all the movies go blank and a "timeout" flashes on the bottom middle of screen then vanishes

**Comment by stantanasi** [comment]
**Two new providers added (SoraStream and SuperStream) on [Release 1.6.1](https://github.com/stantanasi/streamflix/releases/latest)**

**Comment by stantanasi** [comment]
**SFlix and TMDb (ex SoraStream) fixed on [Release 1.6.5](https://github.com/stantanasi/streamflix/releases/latest)**
